docs/_static/niscope_usage.inc

```diff
@@ -10,8 +10,98 @@ The following is a basic example of using the **niscope** module to open a sessi
         session.configure_vertical(range=1.0, coupling=niscope.VerticalCoupling.AC)
         session.configure_horizontal_timing(min_sample_rate=50000000, min_num_pts=1000, ref_position=50.0, num_records=1, enforce_realtime=True)
         with session.initiate():
-            wfm, wfm_infos = session[args.channels].fetch(num_samples=args.length)
-        print('Number of samples acquired: {:,}\n'.format(len(wfm)))
+            waveforms = session[[0,1]].fetch()
+        print('Number of samples acquired: {:,}\n'.format(len(waveforms.wfm)))
+
+The waveform returned from `fetch <http://nimi-python.readthedocs.io/en/master/niscope/functions.html#niscope.Session.fetch>`_ is a 1D list of Python objects
+
+    - Attributes:
+
+        -  **relative_initial_x** the time (in seconds) from the trigger to the first sample in the fetched waveform
+        -  **absolute_initial_x** timestamp (in seconds) of the first fetched sample. This timestamp is comparable between records and acquisitions; devices that do not support this parameter use 0 for this output.
+        -  **x_increment** the time between points in the acquired waveform in seconds
+        -  **actual_samples** the actual number of samples fetched and placed in the waveform array
+        -  **gain** the gain factor of the given channel; useful for scaling binary data with the following formula:
+
+            .. math::
+
+                voltage = binary data * gain factor + offset
+
+        -  **offset** the offset factor of the given channel; useful for scaling binary data with the following formula:
+
+            .. math::
+
+                voltage = binary data * gain factor + offset
+
+        - **wfm** floating point array of samples
+
+    - Such that all record 0 waveforms are first. For example, with a channel list of 0,1, you would have the following index values:
```

Comment:
API question - 
Instead of a list of WaveformInfo, why not return list that contains a dictionary where channels are the key and WaveformInfo the value?

Reply:
This would require parsing the channel list to know what the channels are.

Using `session.channel['0,2-4'].fetch()` is valid, but would have to be parsed to know that the list is `[0,2,3,4]`.

I considered doing this so I could reshape the returned list to be 2D (or whatever you would want to call it), but figured you would push back about re-implementing that parsing.

Reply:
Interesting.
We had a similar conundrum in NI-DCPower, and ended up calling a private function they expose.
It's weird to go this far to provide fancy fetch, but stop short with this. Honestly I'm not sure what the right tradeoff is.

Reply:
Added in #759